prediction_market_agent/agents/safe_guard_agent/safe_api_models/transactions.py

```diff
@@ -136,6 +136,14 @@ class Transaction(BaseModel):
     txHash: str | None = None
 
 
+class TransactionWithMultiSig(Transaction):
+    executionInfo: MultisigExecutionInfo
```

Comment:
Type narrowing the general Transaction into transaction with multisig, so we don't have to if-else it everywhere.